var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -108,10 +159,12 @@ class Trilinos(CMakePackage):
     depends_on('glm')
     depends_on('metis@5:', when='+metis')
     depends_on('suite-sparse', when='+suite-sparse')
+    depends_on('zlib', when="+zlib")
 
     # MPI related dependencies
     depends_on('mpi')
     depends_on('netcdf+mpi')
+    depends_on('parallel-netcdf', when="+pnetcdf@master")
```

Comment:
I see. Yes, I should change that. The main thing with parallel-netcdf is that Tribits only recently added capability to find parallel-netcdf, where previously we had to inject the libraries into the netcdf link line. This should be >master, as I don't think there has been a tagged version since.